src/pages/case-study.js

```diff
@@ -97,3 +96,157 @@ export const allCaseStudies = graphql`
+//           {node.New_Style_Structure.Approach_Section.map((section,i)=>{
+//             return <div key={i}>
+//               <div>index: {i}</div>
+//               <div>approach-section-body: {section.body}</div>
+//               <div>approach-section-title: {section.title}</div>
+//               <div>approach-section-image: {section.image?.absolutePath}</div>
+//               <div>approach-section-image-caption: {section.image_caption}</div>
+//             </div>
+//           })}
+//           {node.New_Style_Structure.Key_Outcome.map((outcome,i)=>{
+//             return <div key={i}>
+//               <div>index: {i}</div>
+//               <div>key-outcome-title: {outcome.title}</div>
+//               <div>key-outcome-description: {outcome.description}</div>
+//               <div>key-outcome-icon: {outcome.icon?.url}</div>
+//             </div>
+//           })}
+//           <div>explore: {node.New_Style_Structure.Explore}</div>
+//         </div>
+//       })}  */}
+//     </GeneralLayout>
+//   );
+// };
+
+// export default CaseStudies
+
+// export const query = graphql`
+//   {
+//     allStrapiCaseStudy {
+//       nodes {
+//         id
+//         Path
+//         Style
+//         Title
+//         Client_Name
+//         New_Style {
+//           Title
+//           Path
+//           Outcome_Text
+//           Explore
+//           Client_Name
+//           Client_Goal_Text
+//           Challenge_Text
+//           Approach_Text
+//           Approach_Image {
+//             url
+//           }
+//           Approach_Section {
+//             Text
+//             Title
+//             Image {
+//               publicURL
+//             }
+//           }
+//           Client_Goal_Bullet {
+//             Text
+//           }
+//           Expertise {
+//             Text
+//           }
+//           Impact_Stat {
+//             Header
+//             Text
+//           }
+//           Outcome {
+//             Caption
+//             Title
+//             Icon {
+//               url
+//             }
+//           }
+//           Preview_Image {
+//             url
+//           }
+//           Quote {
+//             Quote
+//             Source
+//           }
+//           Realated_Titles {
+//             Text
+//           }
+//           Technology {
+//             Text
+//           }
+//         }
+//         Old_Style {
+//           Approach
+//           Project
+//           Website
+//           Background {
+//             Section_One
+//             Section_Two
+//             Title
+//           }
+//           Images {
+//             alternativeText
+//             caption
+//             url
+//           }
+//           Preview_Image {
+//             publicURL
+//           }
+//           Related_Titles {
+//             Text
+//           }
+//           Specs {
+//             Text
+//           }
+//         }
+//       }
+//     }
+//   }
```

Comment:
Let's delete this instead of commenting it out. It can be restored from git history if needed in the future.